frontend/src/app/features/dashboard/task-form/task-form.component.ts

```diff
@@ -36,7 +37,20 @@ export class TaskFormComponent extends BaseComponent implements OnInit {
     }
 
     public getLabel(period: TaskRepeatingPeriod): string {
-        return getPeriodLabel(period);
+        switch (period) {
+        case TaskRepeatingPeriod.None:
+            return 'None';
```

Comment:
If this text is displayed on UI, where is the support of translation? Also, I do not see translations support for errors

Reply:
Fixed